happybase/Hbase.thrift

```diff
@@ -75,7 +75,7 @@ struct ColumnDescriptor {
   6:i32 bloomFilterVectorSize = 0,
   7:i32 bloomFilterNbHashes = 0,
   8:bool blockCacheEnabled = 0,
-  9:i32 timeToLive = -1
+  9:i32 timeToLive = 0x7fffffff
```

Comment:
am i right in assuming that this is a _unmodified_ upstream version of the thrift file?